iBudget/ibudget/views.py

```diff
@@ -34,16 +34,15 @@ def post(self, request):# pylint: disable=no-self-use
         <input type='file' name = 'icon'>
         """
         try:
-            print(request.FILES)
             if request.FILES:
                 pic = request.FILES['icon']
-                if AwsService.check_if_in_bucket(pic):
-                    if AwsService.upload(pic):
-                        pic = str(pic)
-                        url = AwsService.get_image_url(pic)
-                        return HttpResponse(url)
-                    return RESPONSE_500_NO_SUCCESS
-                return RESPONSE_400_INVALID_DATA
+                while AwsService.check_if_in_bucket(pic) != True:
```

Comment:
`while AwsService.check_if_in_bucket(pic) != True:` ->`while not AwsService.check_if_in_bucket(pic):`

Reply:
ok, I will change this too!